### ISSUE TITLE
[BUG] Crash with UE 5.3.2 Game

### ISSUE THREAD

**Issue by armax9** [praydog/UEVR#144]
When I run UE 5.3.2 Game (Wild Life), the game immediately crashes. Attaching the log

[UE-5.3.2-crash-log.zip](https://github.com/praydog/UEVR/files/13978119/UE-5.3.2-crash-log.zip)

**Comment by praydog** [comment]
Please upload your global dir folder too.

**Comment by praydog** [comment]
And identifying the version of UEVR will help. Is it an Actions build? Is it a release build? Do you know what the commit hash is? The issue templates should tell you all of this.

**Comment by praydog** [comment]
Was able to find it in one of the recent Actions builds.

**Comment by praydog** [comment]
Looks related to freezing CVars. Did you freeze any CVars? You might want to clear out your config.

**Comment by armax9** [comment]
The latest Action build Commit [e1b96b1], and 1.03 both crash the game. Sorry for not mentioning it.

I cleaned the UEVR folder, as you proposed. Now injector injects, but the VR doesn't start. Just loading the screen in Oculus. The game doesn't crash, but VR doesn't start. 

You told me about "your debugger". Sorry, I can't find any related info about it. Do you mean VS2022 debugger?

**Comment by armax9** [comment]
SceneView Compatability adds headset movement to the game, but only on PC screen. No picture in the headset.

**Comment by praydog** [comment]
No, I was just trying to load the exact same UEVRBackend.dll you used so I could analyze the crash dump better. I figured that part out now.

But the issue seems related to the CVar system. I was not crashing in this but it is weird that it was crashing you. Which is why I need to know what CVar you may have froze so I can try to replicate the issue.

The other problem you are talking about, VR not starting is probably because the game has 3 VR plugins in the folder. You can rename all of them.

**Comment by armax9** [comment]
I run the game with -nohmd and renamed OpenXR folder. Still no luck. 
Probably game-specific problem.

**Comment by armax9** [comment]
OpenXR doesn't start in VR.
OpenVR crashes the game. 
Commit [https://github.com/praydog/UEVR/commit/e1b96b1ef9f92b7074e63ce01b648b10b2b64dc5],

Attaching the log. 
[OPENVR-CRASH.zip](https://github.com/praydog/UEVR/files/13978725/OPENVR-CRASH.zip)

Game profile is clean. 
[WildLifeC-Win64-Shipping.zip](https://github.com/praydog/UEVR/files/13978741/WildLifeC-Win64-Shipping.zip)
